src/app/content/components/CallToActionPopup/styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import Button from '../../../components/Button';
+import { textRegularStyle } from '../../../components/Typography';
+import { textStyle } from '../../../components/Typography/base';
+import theme from '../../../theme';
+
+// tslint:disable-next-line: variable-name
+export const CTAWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+
+  @media screen and (max-width: 1200px) {
+    display: none;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAContent = styled.div`
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  width: 100%;
+  display flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAText = styled.div``;
+
+// tslint:disable-next-line: variable-name
+export const CTAHeading = styled.h2`
+  ${textStyle}
+  font-size: 3.6rem;
+  line-height: 4rem;
+  color: ${theme.color.text.white};
+  margin: 0;
+  max-width: 570px;
+  overflow: hidden;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAParagraph = styled.p`
+  ${textStyle}
+  font-size: 1.8rem;
+  line-height: 3rem;
+  margin: 1rem 0;
+  color: ${theme.color.text.white};
+  max-width: 570px;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAButtons = styled.div`
+  display: flex;
+  align-items: center;
+  ${Button} {
+    width: 170px;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTATextLink = styled.span`
+  ${textRegularStyle}
+  color: ${theme.color.text.white};
+  margin-left: 1.5rem;
+  a {
+    color: ${theme.color.text.white};
+    text-decoration: underline;
+    margin-left: 0.5rem;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTACloseButton = styled(Times)`
+  position: absolute;
+  top: 40%;
+  right: 3rem;
+  z-index: 3;
+  width: 1.8rem;
+  color: ${theme.color.text.white};
+  cursor: pointer;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAGraphic = styled.img`
+  margin-left: 3rem;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTATopLayer = styled.img`
```

Comment:
for all of these you could do
```suggestion
export const CTATopLayer = styled((props) => <img {...props} src={thesrc} />)`
```
that way you're defining what it looks like all in one place, and the index will just arrange them

Reply:
I had to change `styles.ts` to `styles.tsx`, don't be scared about diff :D